Remotion/Web/Core/Doc/include/Configuration/ResourceRoot.xml

```diff
@@ -15,34 +15,33 @@
  ! You should have received a copy of the GNU Lesser General Public License
  ! along with re-motion; if not, see http://www.gnu.org/licenses.
 -->
-<ResourceConfiguration>
+<ResourceRoot>
   <Class>
     <remarks> 
       See the <see cref="WebConfiguration"/> for details on registering the <c>remotion.web</c> configuration section.
     </remarks>
     <seealso cref="WebConfiguration"/>
     <seealso cref="Remotion.Web.ResourceUrlResolver"/>
     <example>
-      Setting the <c>resources</c> entry's values in the <c>remotion.web</c> configuration section.
+      Configuring the resource root is done by adding a new implementation of
```

Comment:
We can keep the example, but I'm also fine with dropping it and simply adding a one-liner on how the extension point is intended. re-motion has so many IoC services that require customizations on a much more frequent level that having this much doc here feels like overkill. Said another way: realistically, nobody is ever going to see and act on this.